### ISSUE TITLE
wenet/utils/common.py中pad_list函数注释有误或未实现完全

### ISSUE THREAD

**Issue by gengxuelong** [wenet-e2e/wenet#2007]
wenet/utils/common.py中,错误处以=====>标注:
def pad_list(xs: List[torch.Tensor], pad_value: int):
"""Perform padding for the list of tensors.

Args:
xs (List): List of Tensors [(T_1, *), (T_2, *), ..., (T_B, *)]. bug:=================>'*'不应存在,或者说代码未实现 *
pad_value (float): Value for padding.

Returns:
Tensor: Padded tensor (B, Tmax, *). bug: ===================>同上

Examples:
>>> x = [torch.ones(4), torch.ones(2), torch.ones(1)]
>>> x
[tensor([1., 1., 1., 1.]), tensor([1., 1.]), tensor([1.])]
>>> pad_list(x, 0)
tensor([[1., 1., 1., 1.],
[1., 1., 0., 0.],
[1., 0., 0., 0.]])

"""
n_batch = len(xs)
max_len = max([x.size(0) for x in xs])
pad = torch.zeros(n_batch, max_len, dtype=xs[0].dtype, device=xs[0].device)
pad = pad.fill_(pad_value)
for i in range(n_batch):
pad[i, :xs[i].size(0)] = xs[i]

return pad

**Comment by robin1001** [comment]
Yes, good catch! Could you help to fix that?

**Comment by gengxuelong** [comment]
> Yes, good catch! 可以帮忙提 PR 修复一下吗？

当然可以,我已经提交了PR
